sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/ResourceGroupOperations.cs

```diff
@@ -963,5 +969,194 @@ public virtual T UseClientContext<T>(Func<Uri, TokenCredential, ArmClientOptions
         {
             return func(BaseUri, Credential, ClientOptions, Pipeline);
         }
+
+        /// <summary>
+        /// Creates or updates the entire set of tags on a resource or subscription.
+        /// </summary>
+        /// <param name="tags"> The set of tags. </param>
+        /// <param name="cancellationToken"> A token to allow the caller to cancel the call to the service. The default value is <see cref="CancellationToken.None" />. </param>
+        /// <returns> Tags from the specified object. </returns>
+        public virtual Response<TagsResource> CreateOrUpdateAllTags(IDictionary<string, string> tags, CancellationToken cancellationToken = default)
```

Comment:
lets not add the "new functionality" lets only update the existing methods for now

Reply:
Done